### PR TITLE
Add Matrix Room to Readme

### PR DESCRIPTION
**Pull request by lewri**
**Describe what the proposed change does**
- IRC is dead (for now). Long live Matrix?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +80,6 @@ Our [wiki](https://github.com/CorsixTH/CorsixTH/wiki) is a good place to start,
 ## Contact
 
 - Follow us on [Reddit](https://www.reddit.com/r/corsixth), Twitter ([**@CorsixTH**](https://twitter.com/CorsixTH)), and on [Facebook](https://facebook.com/CorsixTH)
-- IRC [#Corsix-TH](https://webchat.freenode.net/#corsix-th) on irc.freenode.net
+- Hit up our [Matrix Room (Discord Bridge)](https://matrix.to/#/#corsixth-general:matrix.org) (web acccess available via [Element.io](https://element.io/get-started))
```
Comment (TheCycoONE) [inline README.md:83]: the web access link is probably not needed, iirc the matrix.to link is pretty informative.

**Comment by TheCycoONE** [comment]
Ping @Alberth289346 @wolfy1339 @MarkL1961

File: README.md (hunk 1 above)
Comment (lewri) [reply README.md:83]:
```suggestion
- Hit up our [Matrix Room (Discord Bridge)](https://matrix.to/#/#corsixth-general:matrix.org)
```

File: README.md [hunk 2]
```diff
@@ -75,6 +80,6 @@ Our [wiki](https://github.com/CorsixTH/CorsixTH/wiki) is a good place to start,
...
 - Join the server on [Discord](https://discord.gg/Mxeztvh)
```
Comment (emorrp1) [inline README.md]:
```suggestion
- Matrix chat via [#corsixth-general:matrix.org](https://matrix.to/#/#corsixth-general:matrix.org) (see also Discord)
- Join the server on [Discord](https://discord.gg/Mxeztvh) (see also Matrix)
- If you would like us to register on another IRC server, please open a new issue
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,12 @@
 ![](http://i.imgur.com/fYp148T.jpg)
+
+#### Important! ####
+On 2021-06-15 freenode removed the corsix-th channel on IRC causing migration off the network. We have added a matrix space bridged to the existing discord server. If you would like us to register on another IRC server, **please open a new issue**.
```
Comment (emorrp1) [inline README.md:4]: I'm just wary of time-limited information, I always forget to go back and remove it when no longer needed.

**Pull request merged**